src/Controller/ExportController.php

```diff
@@ -336,9 +351,27 @@ private function buildXmlProducts(
                         $productEntity->getId()
                     )
                 );
+            } catch (ProductHasCrossSellingCategoryException $e) {
+                $this->logger->warning(
+                    sprintf(
+                        'Product with id %s %s was not exported because it is assigned to cross selling category %s %s',
```

Comment:
Please put the product name and the category breadcrumb in braces. This way it is easier to detect how long the actual name is and also where it actually ends, since a product usually contains multiple words.